packages/elements/src/checkbox/__test__/checkbox.test.js

```diff
@@ -206,6 +206,7 @@ describe('checkbox/Checkbox', () => {
         const onChecked = () => el.dispatchEvent(new Event('tap'));
         setTimeout(onChecked);
         const e = await oneEvent(el, 'checked-changed');
+        await nextFrame();
```

Comment:
I doubt that we need `nextFrame()`, maybe it just need an `elementUpdated()` ?

Reply:
Yes, that is correct.

Reply:
Fixed